web/templates/subscribe_to_more_streams.hbs

```diff
@@ -1,16 +1,16 @@
 {{#if exactly_one_unsubscribed_stream}}
     <a href="#streams/all">
         <i class="fa fa-plus-circle" aria-hidden="true"></i>
-        {{~#tr}}Browse 1 more stream{{/tr~}}
+        {{t "Browse 1 more stream" }}
```

Comment:
Hmm, the `~`s there were whitespace control characters -- not part of the translation but I suspect they were intended to  make selecting this string not select the whitespace around it too.

Reply:
Ah, so probably updating the regex in `makemessages.py` then for these `~` use cases, which was new in those changes?

Reply:
OK fixed the actual bug here and restored the whitespace control characters. Here's the diff in `translations.json`, confirming that these were the only strings affected:
```
--- locale/en/translations.json	2023-09-08 08:35:06.806695411 -0700
+++ /tmp/translations.json	2023-09-08 08:32:19.694787284 -0700
@@ -196,10 +196,8 @@
   "Bot owner": "Bot owner",
   "Bot type": "Bot type",
   "Bots": "Bots",
-  "Browse 1 more stream": "Browse 1 more stream",
   "Browse recent conversations": "Browse recent conversations",
   "Browse streams": "Browse streams",
-  "Browse {can_subscribe_stream_count} more streams": "Browse {can_subscribe_stream_count} more streams",
   "Business": "Business",
   "Busy": "Busy",
   "By deactivating your account, you will be logged out immediately.": "By deactivating your account, you will be logged out immediately.",
```

I still think it's an improvement to switch to `{{t` -- the `#tr` thing is intended for multi-line situations, so I'll merge a version of this commit but with a different commit message.